getting_started/setup_vm/roles/ccf_build/vars/common.yml

```diff
@@ -33,6 +33,6 @@ debs:
 docker_debs:
   - docker-ce-cli
 
-doxygen_ver: "1.9.4"
+doxygen_ver: "1.9.5"
```

Comment:
A new version of doxygen was released concurrently with this PR and the old 1.9.4 install tarball is no longer available.